__tests__/article.test.js

```diff
@@ -63,7 +63,7 @@ describe("04 GET /api/articles/:article_id", () => {
         .expect(200)
         .then(({ body }) => {
           const { article } = body;
-          expect(article).toEqual(output);
+          expect(article).toEqual(expect.objectContaining(output));
```

Comment:
I hope we've learned a valuable lesson about writing flexible tests, so we don't have to go back and change them when we add additional functionality 👀